release_images.yml

```
@@ -93,9 +93,9 @@ release_images:
     version: "1.15.5"
     training:
       device_types: ["gpu"]
-      python_versions: ["py37"]
+      python_versions: ["py36"]
```

We need to release the following example images:
Py36 - GPU-CUDA-10.0
Py37 - GPU-CUDA-10.0

These can be configured by making the following changes:
```suggestion
      python_versions: ["py36", "py37"]
      os_version: "ubuntu18.04"
      cuda_version: "cu100"
      example: True
      disable_sm_tag: False
      force_release: False
```

---

I think last night when TF1.15.5 release was triggered, the Py37 example image was also released. I'm not sure if the example image would have its own tag on GitHub or it didn't get to the tag part because the container wasn't updated.